README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
+across different servers. 
+
+Afterwards, simply start the server via `mscs start [world]` where `world` 
+is the name of the world. 
+
+**Finally, accept the EULA**.
+As of Minecraft version 1.7.10, Mojang requires that users of their software 
+read and agree to their 
+[EULA](https://account.mojang.com/documents/minecraft_eula).  
+After the first time you start the server, you need to modify the `eula.txt` 
+file in your world's folder, changing the value of the `eula` variable from 
+`false` to `true`.
+
+The EULA can be found in `/opt/mscs/worlds/myWorld` where `myWorld` 
+is the name given to the world you created.
+
+After accepting the EULA simply start the server using the same command above, 
+and you're all set!
+
+### Importing an existing world
+Suppose you want to import a world folder named `world` into MSCS, and that you 
+want MSCS to recognize this world by the name "vanillaMC".
+
+1. First, if you don't have one already, create a `worlds` folder in 
+`/opt/mscs/`.
+
+2. Create a new folder **within the `/opt/mscs/worlds/` directory that is the 
+name you want MSCS to use for the world**.
+3. For this example, I chose "vanillaMC". So, I created a new directory
+`vanillaMC` within the `/opt/mscs/worlds` directory, so the path would be 
+`/opt/mscs/worlds/vanillaMC`. 
+
+3. Drag the folder of the world you wish to move into the folder you 
+just created. 
+So I would drag the world `world` into the `vanillaMC` folder. 
+The path of `world` (the actual world folder) would now be 
+`/opt/mscs/worlds/vanillaMC/world/`.
+
+The finished file structure should resemble the following, with a world named 
+`world` and a containing folder name "vanillaMC":
+````
+/opt/mscs/vanillaMC       // The path
+
+world                     // The actual world folder
+server.properties         
+other world files... (in no particular order)
+````
+
+After you've set up the file structure, you now need to create a world entry 
+into MSCS. Do this via:
+
+        mscs create [world] [port] <ip>
+  
+Where `world` is the **name of the containing folder you created** 
+(so it would be "vanillaMC" from the previous example", 
+and `port` is the server port (by default, use 25565).
+`ip` is optional and will be used if you wish to create multiple worlds 
+across different servers. For now, leave it blank.
+
+Afterwards, simply start the server via `mscs start [world]` where `world` 
+is the name of the containing world's folder (again, it would be "vanillaMC" 
+from the last example). 
+Then, accept the EULA, run the start command again, and you're good to go!
+
+As a last note, make sure you check `server-port` and `query.port` in 
+`server.properties` 
+to make sure it does not overlap with other servers created by the MSCS script. 
+Also ensure that `enable-query` is set to `true`.  
+If you do not have `enable-query` and a `query.port` set, 
+you will not be able to check the status of the world with the script.
+
```

Comment:
I don't think that this sequence that you have laid out would actually work.  Did you test it?

If I read the directions correctly, the `server.properties` file is in `/opt/mscs/worlds/vanillaMC/`, and the world data is in `/opt/mscs/worlds/vanillaMC/world`.  However, when you run `mscs create`, it changes the `level-name` value in `server.properties` to `vanillaMC` from `world`, so that when Minecraft starts running it won't find the world data since it expects to find it in `/opt/mscs/worlds/vanillaMC/vanillaMC`.

Making a copy of a world and renaming the world are two different problems.  Renaming a world requires that both the `server.properties` file change and the location of the world data matches that change.  Perhaps I should add a rename command to the script to simplify this.